docs/cloud/programmaticaccess.md

```diff
@@ -12,7 +11,7 @@ Everything that can be done in the user interface can be accomplished via REST a
 
 ## Obtain an auth token
 
-FeatureBase Cloud uses Oauth2.0 for all authorization, so every API call must be accompanied with a valid token. You can get tokens by passing your credentials to https://id.featurebase.com . The below cURL command can be run on any linux-based system:
+FeatureBase Cloud uses Oauth2.0 for all authorization, so every API call must be accompanied with a valid token. You can get tokens by passing your credentials to https://id.featurebase.com. The below cURL command can be run on any linux-based system:
```

Comment:
I'm going to leave this alone and do a rework later.
But in short, the second sentence is a procedure/process so it needs to be under a separate heading.
In fairness to you, I haven't put together any API templates/docs because it was all going to be shifted to redoc.ly.
So if it's remaining in help, I'll put it together soon :)